_content/av/taking-responsibility-for-your-mind_jayasaro.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Taking Responsibility For Your Mind"
+authors:
+  - jayasaro
+external_url: "https://www.abhayagiri.org/talks/4988-taking-responsibility-for-your-mind"
+drive_links:
+  - "https://drive.google.com/file/d/1krDSZIeOAJUtsMHm8O_gfnzUSveTAuz5/view?usp=drive_link"
+formats: [mp3]
```

Comment:
mp3 is the default for `av` works, so you can feel free to include or omit as you like in these cases 😊